src/CallHandler.tsx

```diff
@@ -77,13 +77,18 @@ import ErrorDialog from "./components/views/dialogs/ErrorDialog";
 import WidgetStore from "./stores/WidgetStore";
 import { WidgetMessagingStore } from "./stores/widgets/WidgetMessagingStore";
 import { ElementWidgetActions } from "./stores/widgets/ElementWidgetActions";
+import { MatrixCall, CallErrorCode, CallState, CallType } from "matrix-js-sdk/lib/webrtc/call";
 
-// until we ts-ify the js-sdk voip code
-type Call = any;
+enum AudioId {
```

Comment:
`AudioID` to match conventions (which are informal and awful, so happy to be argued on this)